docs/site/src/pages/reference/concepts/writes.mdx

```diff
@@ -0,0 +1,205 @@
+see the [Python Client](../python-client/write-data) 
+and [TypeScript Client](../typescript-client/write-data) guides.
+
+<Divider.Divider direction="x" />
+
+## Introduction
+
+Writers are used to write telemetry samples to a Synnax cluster. A writer is opened on
+a set of channels, and typically writes to these channels in a continuous stream over
+long periods of time. 
+
+<Divider.Divider direction="x" />
+
+## Transactions
+
+Each writer governs an atomic transaction against the Synnax cluster. To begin a 
+transaction, we open a new writer and start writing data frames using a file like 
+interface. After we're done writing, we commit the writer to the database, making it
+available for reading. Synnax transactions are unique in that they can be committed 
+**multiple** times, where each new commit extends the data written in the previous commit. 
+After the final commit, the user closes the writer, at which point the cluster releases 
+all write related resources and the transaction is complete.
+
+### Open Phase
+
+To start a new transaction, we need to provide the following information.
+
+<Table>
+   <thead>
+      <tr>
+         <th>Argument</th>
+         <th>Description</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Channels</td>
+         <td>The channels we're interested in writing to.</td>
+      </tr> 
+      <tr>
+         <td>Timestamp</td>
+         <td>The timestamp marking the time at or just before the first written sample was recorded.</td>
+      </tr>
+   </tbody>
+</Table>
+
+Synnax will use these arguments to validate a few things before opening the writer:
+
+1. That all channels exist in the Synnax cluster.
+2. That no existing data overlaps with the provided timestamp.
+
+#### Optional Arguments
+
+There are also a few optional arguments we can provide when opening a writer:
+
+<Table>
+   <thead>
+      <tr>
+         <th>Argument</th>
+         <th>Description</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td><a href="#persistencestreaming-modes">Mode</a></td>
+         <td>The persistence/streaming mode of the writer. This can be persist only, stream only, or stream + persist.</td>
+      </tr> 
+      <tr>
+         <td><a href="#dynamic-control---write-authorities">Authority</a></td>
+         <td>The authority to set for the writer. This is used for dynamic control handoff between multiple writers.</td>
+      </tr>
+      <tr>
+         <td>Auto-Commit</td>
+         <td>Whether writes should be immediately committed to the Synnax Cluster.</td>
+      </tr>
+   </tbody>
+</Table>
+
+### Write Phase
+
+After all opening validation checks pass, the transaction enters the write phase. 
+During this period, a user can write continuous frames of data to the transaction. These
+frames must be vertical (i.e. all channels must have a single series specified),
+and all series must have the same number of samples. You can read more about
+frame alignment here.
```

Comment:
maybe say below instead of here - thought it was referring to a missing link

Reply:
Removed this, and added linear issue https://linear.app/synnaxlabs/issue/SY-1058/document-frames-series-concepts-again to update the frames and series documentation.